corehq/apps/userreports/tasks.py

```diff
@@ -276,17 +277,25 @@ def save_document(doc_ids):
             indicator = indicator_by_doc_id[doc['_id']]
 
             eval_context = EvaluationContext(doc)
-            try:
-                for config_id in indicator.indicator_config_ids:
+            for config_id in indicator.indicator_config_ids:
+                adapter = None
+                try:
                     config = _get_config(config_id)
                     adapter = get_indicator_adapter(config, can_handle_laboratory=True)
-                    adapter.best_effort_save(doc, eval_context)
```

Comment:
Because I want to manually handle the exceptions. It doesn't matter if es or couch have an error as it will retry later, but other errors I want to handle